tests/env/testdata/fake_jwt.go

```diff
@@ -14,13 +14,18 @@
 
 package testdata
 
+import (
+	confpb "google.golang.org/genproto/googleapis/api/serviceconfig"
+)
+
 type ProviderConfig struct {
 	Id               string
 	Issuer           string
 	Keys             string
 	IsInvalid        bool   // If invalid, then a server that always returns 503 is started up
 	IsNonexistent    bool   // If non-existent, then a server is not started up
 	HardcodedJwksUri string // This needs to be set for non-existent, since the URL cannot be derived
+	JwtLocations     []*confpb.JwtLocation
```

Comment:
Shouldn't this be set in the service config? I'm confused how this is applied to the service config

Reply:
This is per provider http://google3/google/api/auth.proto?l=320&rcl=297378306